src/DockerContainer.php

```diff
@@ -46,6 +46,8 @@ class DockerContainer
 
     public array $commands = [];
 
+    protected float $startCommandTimeout = 60;
```

Comment:
Where is this value actually being used?

Reply:
Getters/setters here:

https://github.com/spatie/docker/pull/53/commits/18ee5a3c771cc698f4845e61cecbfaf47ece61cf#diff-92b49ac2202fd26837653c9c7f9391e0f07cfc124aa7a85bbe753acf07ba7470R292

then used here:

https://github.com/spatie/docker/pull/53/commits/18ee5a3c771cc698f4845e61cecbfaf47ece61cf#diff-92b49ac2202fd26837653c9c7f9391e0f07cfc124aa7a85bbe753acf07ba7470R275